pkg/maps/policymap/statsmap.go

```diff
@@ -0,0 +1,148 @@
+// SPDX-License-Identifier: Apache-2.0
+// Copyright Authors of Cilium
+
+package policymap
+
+import (
+	"errors"
+	"fmt"
+
+	"github.com/cilium/ebpf"
+	"golang.org/x/sys/unix"
+
+	"github.com/cilium/cilium/pkg/bpf"
+	"github.com/cilium/cilium/pkg/byteorder"
+	"github.com/cilium/cilium/pkg/logging/logfields"
+)
+
+const (
+	// Name is the canonical name for the policy stats map on the filesystem.
+	// Note: There is no underscore between 'policy' and 'stats' as that would confuse tools
+	// trying to parse an endpoint ID from 'stats'.
+	StatsMapName = "cilium_policystats"
+)
+
+var (
+	// MaxEntries is the maximum number of keys that can be present in the
+	// policy stats map.
+	// This number is rounded down to the closest multiple of 'ebpf.PossibleCPU()' to avoid map
+	// size mismatch and endless repinning due to a size mismatch.
+	MaxStatsEntries = 65536
+)
+
+// PolicyStatsMap maps endpoint IDs to the fd for the program which
+// implements its policy.
+type StatsMap struct {
```

Comment:
If I understand correctly, this is a new map - for new code, I'd expect a map cell (like the examples in `pkg/maps/cells.go`). Is there a reason this isn't a cell?

Reply:
No real reason, added a new commit with a Cell that handles the new map and the config of the Endpoint's policymap.

Reply:
OK, the real reason was that it was a ton of work. Done.

Reply:
Thanks for doing it anyway.